### DISCUSSION TITLE
Request to add a Swift v1 tutorial or example

### DISCUSSION THREAD

**Discussion by KakaoTocs** [WalletConnect/walletconnect-monorepo#712] [Developer Support]
I am a native iOS developer.
I don't know deeply about web3 and Ethereum systems.

I just want to get the user's wallet address from the iOS native app.
However, MetaMask connections will only allow users who have been verified as owners.

Is there a simpler example or tutorial?

The current example is confusing because it is not linked to the actual wallet, but only connected between Client and Server apps in the sample project.

**Comment by chadyj** [comment]
Thanks for the feedback @KakaoTocs.

We plan on improving our docs and sample code with simple use cases like you describe.  In the meantime, you can check out our getting started guide https://medium.com/walletconnect/beginner-guide-to-walletconnect-v2-0-for-swift-developers-4534b0975218

**Reply by KakaoTocs** [reply]
Thank you so much for answering.
I'll try it after reading the medium article.
Additional , It's not related to this project, but can I be sure that the two examples of wallet connect v1 work normally?

**Reply by KakaoTocs** [reply]
It refers to these projects.
* https://github.com/WalletConnect/swift-walletconnect-lib/tree/master/Example
* https://github.com/WalletConnect/WalletConnectSwift-Example/tree/main/ExampleApps

**Reply by chadyj** [reply]
> Additional , It's not related to this project, but can I be sure that the two examples of wallet connect v1 work normally?

v1 should work normally, and is used in production in various apps.  But if you are having issues you can post here.

Also FYI, v2 is in development and isn't widely deployed yet, so if you are looking to integrate with wallets today, you should use v1.

**Reply by KakaoTocs** [reply]
* https://github.com/WalletConnect/WalletConnectSwift-Example
Link to the recorded video
* https://drive.google.com/file/d/1G03w40qcHEqpMjCtFkeCHW5fgCkxjx8b/view?usp=sharing

I downloaded this project and only executed it.
(I only selected Signing -> Team to operate on the device.)
Did I make a mistake?

**Reply by KakaoTocs** [reply]
@chadyj  I want to check if you saw this comment.

**Reply by chadyj** [reply]
@KakaoTocs Right now the team is focused on V2 examples and documentation. If you have any issues with that we would love to help you out.

**Reply by KakaoTocs** [reply]
@chadyj 
I want you to help me.
I want to know what I did wrong when build the project.

**Reply by jason18401** [reply]
@KakaoTocs any luck on building the project?

**Reply by KakaoTocs** [reply]
@jason18401 No, it's currently stopped.

**Reply by KakaoTocs** [reply]
@chadyj Can I get help from your team?

**Reply by KakaoTocs** [reply]
I recently tried again and succeeded.
Thank you.
